### DISCUSSION TITLE
Is npm i react-native-elements deprecated?

### DISCUSSION THREAD

**Discussion by russmenum** [react-native-elements/react-native-elements#3637] [Q&A]
I have an old app I am rebuilding and it is using ```react-native-elements```

NPM says ```npm i react-native-elements``` [here](https://www.npmjs.com/package/react-native-elements) but that seems like an old ver?

Links to this repo which says ```npm install @rneui/themed @rneui/base```

It looks like ```import { Input, Icon } from "react-native-elements";``` is all that is being used.

But is that now from ```"@rneui/themed"```? Is this a fork? not seeing clear doc on how we get from v3.4.2 to v4.x with the diferent NPM

**Comment by arpitBhalla** [comment]
Refer [Migration Guide](https://reactnativeelements.com/blog/2022/05/15/rneui-migration-guide). It will not be deprecated until v4 is released as stable.

**Reply by russmenum** [reply]
> Refer [Migration Guide](https://reactnativeelements.com/blog/2022/05/15/rneui-migration-guide). It will not be deprecated until v4 is released as stable.

Let me ask this another way, for maintainability is it better to ```npm i react-native-elements``` or ```npm install @rneui/themed @rneui/base``` at this time? 

Because I am not seeing a timeline here if there is one, and there is pro/con to using a deprecated or soon-to-be deprecated.

**Reply by arpitBhalla** [reply]
I believe `npm install @rneui/themed @rneui/base`
